winit-core/src/event.rs

```diff
@@ -438,7 +438,7 @@ pub enum PointerKind {
     ///
     /// **macOS:** Unsupported.
     Touch(FingerId),
-    Stylus(StylusTool),
+    Tool(ToolKind),
```

Comment:
Prefix everything with `Tablet` please, since it's not clear what tool otherwise(As I initially requested).

Reply:
My thought was that after a quick search, i didn't find much about how other platforms handle that "tablet pad" buttons and thought I may need to combine tool and pad buttons, but even then a tablet prefix would be smart so yeah I'm gonna change that when I'm back at my pc.